hack/cosign-verify-image.sh

```diff
@@ -40,12 +40,27 @@ logical_image="${1}"
 physical_image="${2}"
 
 echo -ne "Validating ${logical_image} ... "
-for key_url in "${HPE_OCI_SIGNING_KEYS[@]}"; do
-    key=$(basename "${key_url}")
-    if cosign verify --key "${BUILDDIR}/security/keys/oci/${key}" --insecure-ignore-tlog --insecure-ignore-sct "${physical_image}"  2>/dev/null 1>/dev/null; then
+tmpdir=$(mktemp -d)
+trap "rm -rf ${tmpdir}" EXIT
```

Comment:
I prefer to have trap call a function.

- Shell variables are expanded when the function is called, rather than at the time the trap is set.
   - This allows the trap function to be put in place prior to creating things that need to be cleaned up.
- It's much easier to add additional cleanup

```
cleanup() {
    [ -n "${tmpdir}" ] && rm -rf ${tmpdir}
}
trap cleanup EXIT
tmpdir=$(mktemp -d)
```

Reply:
Done.